Silicon/CommonSocPkg/Include/Library/RegisterAccess.h

```diff
@@ -0,0 +1,289 @@
+/** @file
+  Header file for register access.
+
+  Copyright (c) 2020 - 2023, Intel Corporation. All rights reserved.<BR>
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+
+**/
+
+#ifndef _REGISTER_ACCESS_H_
+#define _REGISTER_ACCESS_H_
```

Comment:
Each header file under Include\Library is one library interface. For the newly added header files, only Gpiov2Lib.h and R2sbAccessLib.h are good to put to here. Others might put to other places (e.g. add to Library header file if they could be used by other modules, or add to library implementation if they are not used by other modules.